util/environment/environment_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
Tests can probably be made multi-platform by using `os.PathListSeparator`, just like you did in `environment.go`.